### ISSUE TITLE
[Accessibility] [DropDown] Aria check fail for aria-required-children

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#1076]
The drop down component fails accessibility validation for aria-required-children.

Details:
https://dequeuniversity.com/rules/axe/2.1/aria-required-children

Error displayed:
![image](https://cloud.githubusercontent.com/assets/11937034/23207969/3eafd898-f919-11e6-9583-94c76a57c845.png)
